src/applications/static-pages/i18Content.json

```diff
@@ -0,0 +1,4 @@
+{
+  "en": { "linkTitle": "Read this page in English", "langToggleLink": "coronavirus-veteran-frequently-asked-questions" },
+  "es": { "linkTitle": "Leer esta página en Español", "langToggleLink": "coronavirus-veteran-frequently-asked-questions-esp" }
+}
```

Comment:
Is the intent for this file to accommodate more pages and links in the future? The naming and location of the file suggests general use, but the structure is applicable to a very specific page.

If it's only ever going to be single-use, I would not split into a separate file and would instead keep the object local to the module that uses it.

Reply:
Similarly, the naming of the translation module also suggests a general i18n solution, but it's specific to a page. I think it would benefit from some more clarity through naming and/or comments.

Reply:
yeah, its not a generic solution so i moved the object into the associated js file as suggested.